src/sage/schemes/elliptic_curves/kraus.py

```diff
@@ -582,18 +589,18 @@ def check_Kraus_local_2(c4, c6, P, a1=None, assume_nonsingular=False):
 
     - ``P`` -- a prime ideal of the number field which divides 2
 
-    - ``a1`` -- an integral elements of a number field, or None (default)
+    - ``a1`` -- an integral elements of a number field, or ``None`` (default)
 
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
-      check for integrality and nosingularity.
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Either (False, 0, 0) if Kraus's conditions fail, or (True, a1,
-    a3) if they pass, in which case the elliptic curve which is the
+    Either ``(False, 0, 0)`` if Kraus's conditions fail, or ``(True, a1,
+    a3)`` if they pass, in which case the elliptic curve which is the
     (a1**2/12,a1/2,a3/2)-transform of [0,0,0,-c4/48,-c6/864] is
```

Comment:
Same as the above, wont keep mentioning it, but these arrays of values feel as though they should be in double backticks to me.